<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="mainWindowLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionFileOpen_O"/>
    <addaction name="actionFileSave_S"/>
    <addaction name="actionFileSaveAs_A"/>
   </widget>
   <widget class="QMenu" name="menuBuild_B">
    <property name="title">
     <string>构建(&amp;B)</string>
    </property>
    <addaction name="actionBuildCompile_C"/>
    <addaction name="actionBuildSettings_S"/>
   </widget>
   <widget class="QMenu" name="menuProject_P">
    <property name="title">
     <string>项目(&amp;P)</string>
    </property>
    <addaction name="actionNewProject_N"/>
    <addaction name="actionOpenProject_O"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSettings_S">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="actionSettingsFont_F"/>
    <addaction name="actionSettingsColor_C"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuBuild_B"/>
   <addaction name="menuProject_P"/>
   <addaction name="menuSettings_S"/>
  </widget>
  <action name="actionFileOpen_O">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileSave_S">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileSaveAs_A">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionBuildCompile_C">
   <property name="text">
    <string>编译并运行(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionBuildSettings_S">
   <property name="text">
    <string>配置编译选项(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionNewProject_N">
   <property name="text">
    <string>新建项目(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="action_H">
   <property name="text">
    <string>添加头文件(&amp;H)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>添加源文件(&amp;C)</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>添加资源文件(&amp;R)</string>
   </property>
  </action>
  <action name="action_D">
   <property name="text">
    <string>删除(&amp;D)</string>
   </property>
  </action>
  <action name="actionOpenProject_O">
   <property name="text">
    <string>打开项目(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionAddProjectFile_A">
   <property name="text">
    <string>添加(&amp;A)</string>
   </property>
  </action>
  <action name="actionSettingsFont_F">
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
  </action>
  <action name="actionSettingsColor_C">
   <property name="text">
    <string>颜色(&amp;C)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
